docs/windows.md

```diff
@@ -0,0 +1,191 @@
+# Windows Support
+
+Starting with the release version [0.10.0](https://github.com/cloud-hypervisor/cloud-hypervisor/releases/tag/v0.10.0), Cloud Hypervisor has full Windows guest support.
+
+Requirements
+
+- Host with KVM enabled 
+- [UEFI](uefi.md) capable guest
+- Guest with VirtIO integrated drivers
+
+At the current stage, only UEFI capable Windows images are supported. This implies the presence of the OVMF firmware during the Windows installation and in any subsequent usage. BIOS boot is not supported.
+
+The subsequent sections will tell, in detail, how to prepare an appropriate Windows image.
+
+## Image Preparation
+
+### Installation using the stock Windows ISO
+
+Prerequisites
+
+- QEMU
+- Windows installation ISO
+- [VirtIO driver ISO](https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/stable-virtio/)
+- Suitable [OVMF](uefi.md) firmware 
+- Enough disk space
+
+This step currently requires QEMU to install Windows onto the guest. QEMU is only used at the preparation stage, the resulting image is then fully functional with Cloud Hypervisor.
+
+Parametrizing several command parts, these will be used in the follow up sections as well.
+
+```shell
+IMG_FILE=windows-disk.qcow
+WIN_ISO_FILE=en_windows_server_version_2004_updated_may_2020_x64_dvd_1e7f1cfa.iso
+VIRTIO_ISO_FILE=virtio-win-0.1.185.iso
+OVMF_DIR=./FV
+```
+
+Create an empty image file, `qcow` or `raw` is supported.
+```shell
+qemu-img create -f qcow2 $IMG_FILE 30G
+```
+
+Begin the Windows installation process under QEMU
+```shell
+qemu-system-x86_64 -machine q35,accel=kvm \
+	-cpu host \
+	-m 4G \
+	-bios ./$OVMF_DIR/OVMF_CODE.fd \
+	-cdrom ./$WIN_ISO_FILE \
+	-drive file=./$VIRTIO_ISO_FILE,index=0,media=cdrom
+	-drive if=none,id=root,file=./$IMG_FILE \
+	-device virtio-blk-pci,drive=root,disable-legacy=on \
+	-device virtio-net-pci,netdev=mynet0,disable-legacy=on \
+	-netdev user,id=mynet0 \
+	-vga std
+```
+
+Before the installation can proceed, point the Windows installation program to the VirtIO disk and install the necessary storage controller drivers. After that, the attached hard drive will become visible and the actual installation can commence.
+
+After the installation has completed, proceed further to the configuration section. QEMU will be needed at least once more to enable SAC.
+
+### Integrating VirtIO drivers into a Windows ISO
+
+TODO
```

Comment:
I think you need to give instructions on how to install the network driver

Reply:
There are basically two methods

- Have separate Windows ISO and VirtIO driver ISO. This is what is documented currently.
- Integrate VirtIO stuff into the Windows ISO. Such a prepared Windows ISO will be self sufficient as the VirtIO drivers will be available automatically.

The second method doesn't have to be mentioned, actually But i planned to add some instructions so reserved the space. It all depends on the use case. The second case can be useful for the unattended Windows install.


Regarding the network drivers - once pointed to the driver ISO, Windows will pull them when it comes to the network configuration. It happens automaticallly because the installation program records the path to the ISO and will scan it if it finds some device that is missing drivers.

That's how i saw it happening, no extra steps was needed to install the network driver. If some behavior changes, the procedure of installing drivers is actually always same - the unknown device will be shown in the device manager, one will need the driver iso or exe to install it. 

Thanks.

Reply:
I had to manually install the virtio net drivers in the guest (via Device Manager)

Reply:
ACK. I've added now a section wrt device drivers. Underlined the network card specifically.

Thanks.

Reply:
If you're planning to detail this section later (follow up PR), then please remove the section as I don't really want to keep `TODO` in it.